test/com/amazon/ion/impl/OutputStreamWriterTestCase.java

```diff
@@ -179,7 +179,6 @@ private void testFlushing()
         // Try flushing when there's just a pending annotation.
         iw.addTypeAnnotation("fred_1");
         iw.flush();
-        checkFlushed(true);
```

Comment:
This was a useless assertion. The behavior when flushing at the top level with an annotation set but no value written is not defined; there's not need to require IonWriter implementations to actually call `flush()` on the underlying `OutputStream` in this case.